app/src/main/java/com/example/marvel/presentation/ComicsActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.marvel.presentation
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.lifecycle.Observer
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.marvel.R
+import com.example.marvel.presentation.epoxy.ComicsController
+import kotlinx.android.synthetic.main.activity_comics.*
+import org.koin.android.ext.android.inject
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class ComicsActivity : AppCompatActivity() {
+
+    private val comicsViewModel: ComicsViewModel by viewModel()
+    private val comicsController: ComicsController by inject()
```

Comment:
Seguindo o comentário que fiz sobre injetar esse controller (ou não injeta-lo), minha sugestão seria a seguinte:
```suggestion
    private val comicsController: ComicsController by lazy { 
        ComicsController()   
    }
```